models/api.model.js

```diff
@@ -0,0 +1,3 @@
+const endpoints = require('../endpoints.json')
```

Comment:
we dont need a model for this one :) we can just send the endpoints back in the controller :)